po/es.po

```diff
@@ -0,0 +1,175 @@
+# Spanish translation for Gcolor3
+# Copyright (C) 2013-2016 Jente Hidskes <hjdskes@gmail.com>
```

Comment:
Shouldn't this be you, in 2018? ;)

Reply:
I guess not. This copyright statement is related to you, the PACKAGE COPYRIGHT HOLDER.

Below the next line, my name is added as a translator.

But probably, the year for your (C) must be changed, right?

**PS**. Glibc is a good example: https://sourceware.org/git/?p=glibc.git;a=blob;f=po/es.po;hb=HEAD

Reply:
Alright, I'll merge this then and fix those lines in all .po files later.